Assets/Scripts/Characters/Humans/Hero.cs

```diff
@@ -240,6 +240,9 @@ private HumanState state
         public Rigidbody Rigidbody { get; protected set; }
         public SmoothSyncMovement SmoothSync { get; protected set; }
 
+        private Bullet.Source LeftSource => useGun ? Bullet.Source.GunLeft : Bullet.Source.BeltLeft;
```

Comment:
Only really used in the two `LaunchRope` methods, so I could get rid of these.

Reply:
If it's only used within LaunchRope, then convert it into a variable.